ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "ipi/aws/ovn/ipi-aws-ovn-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		],
+		"reviewers": [
+			"aravindhp",
```

Comment:
@aravindhp @ravisantoshgudimetla are you ok with taking excluslve ownership here, or should this list be expanded?

Reply:
I suggest removing @ravisantoshgudimetla and adding @sebsoto and @JacobTanenbaum.

Reply:
Done